types/entities.go

```diff
@@ -13,13 +13,6 @@ import (
 // the Entity (it must be the same as the UID within the Entity itself.)
 type Entities map[EntityUID]*Entity
 
-// An Entity defines the parents and attributes for an EntityUID.
```

Comment:
I extracted this because I always have trouble remembering where the heck this struct is defined.